webapp/channels/src/packages/mattermost-redux/src/actions/teams.ts

```diff
@@ -66,9 +66,7 @@ export function selectTeam(team: Team | Team['id']) {
 export function getMyTeams() {
     return bindClientFunc({
         clientFunc: Client4.getMyTeams,
-        onRequest: TeamTypes.MY_TEAMS_REQUEST,
         onSuccess: [TeamTypes.RECEIVED_TEAMS_LIST, TeamTypes.MY_TEAMS_SUCCESS],
-        onFailure: TeamTypes.MY_TEAMS_FAILURE,
```

Comment:
Can you also remove `MY_TEAMS_REQUEST`, `MY_TEAMS_SUCCESS`, and `MY_TEAMS_FAILURE` as well as the corresponding reducer for this?

Reply:
Removed